Source/Optional.swift

```diff
@@ -59,6 +59,36 @@ public func -<< <T, U>(@noescape f: T -> U?, a: T?) -> U? {
 }
 
 /**
+    compose two functions that produce optional values, from left to right
+
+    - If the result of the first function is `.None`, the second function will not be inoked and this will return `.None`
+    - If the result of the first function is `.Some`, the value is unwrapped and passed to the second function which may return `.None`
+
+    - parameter f: A transformation function from type `A` to type `Optional<B>`
+    - parameter g: A transformation function from type `B` to type `Optional<C>`
+
+    - returns: A function from type `A` to type `Optional<C>`
+*/
+public func >-> <A, B, C>(f: A -> B?, g: B -> C?) -> A -> C? {
+    return { x in f(x) >>- g }
+}
+
+/**
+    compose two functions that produce optional values, from right to left
+
+    - If the result of the first function is `.None`, the second function will not be inoked and this will return `.None`
+    - If the result of the first function is `.Some`, the value is unwrapped and passed to the second function which may return `.None`
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 135 characters (line_length)